src/consensus/replay_tower.zig

```diff
@@ -135,7 +134,7 @@ pub const ReplayTower = struct {
         node_pubkey: Pubkey,
         vote_account_pubkey: Pubkey,
         fork_root: Slot,
-        accounts_db: *AccountsDB,
+        accounts_db: sig.accounts_db.AccountReader,
```

Comment:
nit:

```suggestion
        accounts_reader: AccountReader,
```

With AccountReader aliased properly at the top of the file

Reply:
I'm not convinced the alias would be that helpful; it is used exactly once in this entire file.
55a708f174152d359d406ff5ed95b41c0b8c30ac